app/src/main/res/values/strings.xml

```diff
@@ -972,8 +972,8 @@
     <string name="onboarding_whats_new_description">Have questions about the redesigned %s? Want to know what’s changed?</string>
     <!-- text for underlined clickable link that is part of "what's new" onboarding card description that links to an FAQ -->
     <string name="onboarding_whats_new_description_linktext">Get answers here</string>
-    <!-- text for the firefox account onboarding card header -->
-    <string name="onboarding_firefox_account_header">Start syncing bookmarks, passwords, and more with your Firefox account.</string>
+    <!-- text for the Firefox account onboarding sign in card header -->
+    <string name="onboarding_account_sign_in_header">Start syncing bookmarks, passwords, and more with your Firefox account.</string>
```

Comment:
@eliserichards did you and @ekager already talk about this? We should not be deleting strings ids, because that means we won't be able to uplift strings. Instead, move them to the bottom, let's add a "deprecated strings ids" section.